core/src/test/scala/unit/kafka/log/LogManagerTest.scala

```diff
@@ -136,7 +136,10 @@ class LogManagerTest {
     assertEquals(1, logManager.liveLogDirs.size)
     val logFile = new File(logDir, name + "-0")
     assertTrue(logFile.exists)
-    assertFalse(logManager.directoryId(logFile.getParent).equals(DirectoryId.UNASSIGNED))
```

Comment:
The original assertion is comparing `Option[Uuid]` with `Uuid`, so it always be false. This makes the assertion look a bit off or misleading.